clone/naverclone.css

```
@@ -0,0 +1,179 @@
+* {
+  box-sizing: border-box;
+}
+
+.blind {
+  position: absolute;
+  clip: rect(0 0 0 0);
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+}
+
+header {
+  position: relative;
+}
+
+nav {
+  position: relative;
+}
+
+div#header-banner {
+  width: 1130px;
+  height: 160px;
+  margin: 0 auto;
+  position: relative;
+  border-bottom: 1px solid #e4e8eb;
+} 
+
+div#header-banner > h1 {
+  display:inline-block
```

```suggestion
  display: inline-block;
```

가끔씩 컨벤션 안맞는것들이 보이는데 뭐 실수인것 같으니까! 요기만 체크하겠습니당